### PR TITLE
xlog: New port.

### PR DESCRIPTION
**Pull request by kpreid**

### CODE REVIEW

File: comms/xlog/Portfile [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+homepage            http://www.nongnu.org/xlog/
+master_sites        https://download.savannah.gnu.org/releases/xlog/
+
+checksums           rmd160  a1a78a90c74d7f73ea680fe80a324eaea3e36e95 \
+                    sha256  0f22ebae56f7d0ef0cac6ae7047ec3fe6b330dc68dd7b25ed2c23fdc1c52a10f
+
+patchfiles          patch-configure.diff
+
+depends_lib         port:hamlib \
+                    port:pkgconfig \
```
Comment (neverpanic) [inline comms/xlog/Portfile]: Unless this port needs `pkgconfig` at runtime, this should probably be `depends_build port:pkgconfig`.

File: comms/xlog/Portfile [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+PortSystem          1.0
+
+name                xlog
+version             2.0.14beta4
+
+categories          comms
+license             GPL-3
+platforms           darwin
+maintainers         nomaintainer
```
Comment (kpreid) [inline comms/xlog/Portfile]: I do not feel I am currently able to take the responsibility. In the event that I get myself more organized in the future, I will take maintainership of the ports that I have contributed before then.

File: comms/xlog/Portfile [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                xlog
+version             2.0.14beta4
+
+categories          comms
+license             GPL-3
```
Comment (kpreid) [inline comms/xlog/Portfile]: Done.

**Comment by kpreid** [comment]
Made requested changes. GitHub seems slightly unhappy that I (force) pushed a single revised commit; should I have done something else?

**Comment by neverpanic** [comment]
No, that's fine, it's exactly what I would have expected you to do.

**Pull request merged**